website/borrel/views.py

```diff
@@ -1,13 +1,17 @@
+from datetime import timedelta
```

Comment:
Doesn't Django time zone have time delta?

Reply:
Yes but we are using `time delta` from `datetime` in all other places in TOSTI.